### DISCUSSION TITLE
Asetus korkealle hinnalle

### DISCUSSION THREAD

**Discussion by sahkolasse** [Netgalleria/arska-node#38] [Q&A] [answered]
Moikka

Haluaisin säännön kalliita tunteja varten mutta niin että jos niitä on monta putkeen niin olisi aktiivinen vaikka ensimmäiset kaksi tuntia ja sitten tippuisi pois päältä, onnistuuko tällaisen tekeminen?

Lisänä vielä että voiko sen tehdä niin että jos on pidempi jakso kalliita tunteja niin olisi aktiivinen kaksi ensimmäistä tuntia, sitten tunnin poissa ja sitten taas katsoi uudestaan?

Kiitos avusta jo ennakkoon

Lasse

**Comment by Olli69** [comment, marked as answer]
Moi,
Sääntö  joka pitää kanavan päällä vuorokauden kaksi kalleinta tuntia (ranking 23,24): 
<img width="473" alt="image" src="https://github.com/Netgalleria/arska-node/assets/1752838/150ff396-eb96-45a2-8ae4-5c19f7fa3210">
Vastaavasti jos haluat esim. vaikka (aamu)päivä blokin kaksi kalleinta tuntia (ranking lohkon sisällää 7 ja 8):
<img width="473" alt="image" src="https://github.com/Netgalleria/arska-node/assets/1752838/d4b00b16-aa72-4944-a799-a1fc67ce6fef">

En ole ihan varma tarkoititko tätä, mutta kalleimmat saat tosiaan tekemällä hintarankingille ehdot, jotka täyttyvät vain korkeimmilla ranking-arvoilla.

Olli
